src/start_menu/main_settings/speed.cpp

```diff
@@ -0,0 +1,9 @@
+#include "speed.h"
+#include "../../utils/utils.h"
+#include <iostream>
+
+void Speed(){
+	std::cout << "Give me the time you want between each rendered "
+				 "character (in ms): ";
```

Comment:
First write the commands in single line, then format them.

Reply:
done